site/src/components/layout.js

```diff
@@ -202,9 +206,12 @@ const Layout = ({ children, pageContext }) => {
           },
         ]}
       />
+      {!pageSlug.includes('/cookie-settings') && (
+        <CookieModal contentSource={{ ...getCookieConsent(language, setLanguage), focusTargetSelector: headerId }} />
```

Comment:
Clicking 'esc' when modal is focused does not seem to work in a way that the modal should be minimised and the main page header should be focused instead. So something is not working right 🤔